src/benchmark_environments/util.py

```diff
@@ -1,4 +1,7 @@
 """Miscellaneous utilities."""
```

Comment:
Nit: newline after module docstring.

(I could be persuaded away from this given it doesn't seem to be PEP8/PEP257, but then we should change the other files for consistency.)